calico_containers/pycalico/util.py

```diff
@@ -150,3 +150,44 @@ def validate_icmp_type(icmp_type):
     except ValueError:
         valid = False
     return valid
+
+def validate_hostname_port(hostname_port):
+    """
+    Validate the hostname and port format.  (<HOSTNAME>:<PORT>)
+    An IPv4 address is a valid hostname.
+
+    :return: Boolean: True if valid, False if invalid
+    """
+    # Should contain a single ":" separating hostname and port
+    hostname_port = hostname_port.split(":")
```

Comment:
Any possibility of something that is not a string being passed here?  Do we need to assert that `hostname_port` is a string before we try to split it?